dynamic_programming/number_coin_change/number_coin_change_test.go

```diff
@@ -0,0 +1,36 @@
+package main
```

Comment:
add small description about the code at the top.

Reply:
For test file, what kind of description you need, it standard format to check output of any func. Can you elaborate on it ?

Reply:
With description, I meant what the code does or what a particular program is about. For eg: A line like **program to check the palindrome**